packages/kms/src/index.ts

```diff
@@ -0,0 +1,61 @@
+import { Middleware, MiddlewareCallback, MiddlewareContext } from 'steveo';
+import AWS from 'aws-sdk';
+
+export class KMSMiddleware implements Middleware {
+  client: AWS.KMS;
+
+  keyId: string;
+
+  constructor(keyId: string) {
+    this.client = new AWS.KMS();
+    this.keyId = keyId;
+  }
+
+  public async publish<Ctx = any, C = MiddlewareCallback>(
+    context: MiddlewareContext<Ctx>,
+    _next: C
+  ) {
+    context.payload = await this.encrypt(context.payload);
+  }
+
+  private async encrypt<T = any>(data: T): Promise<string> {
```

Comment:
I'd like us to move to banning 'any' and instead using the more idiomatic 'unknown' in this case.... but in this case i'm not sure why it's needed as a generic type is unknown